### DISCUSSION TITLE
Add a banner telling the user they're not looking at the latest version

### DISCUSSION THREAD

**Discussion by jaskij** [rust-lang/crates.io#10235] [Feature Requests]
Quite often, I'd look up a crate in a search engine, and get linked an outdated version for no good reason. From there, getting to the latest version requires:

- realizing it happened
- clicking "Versions"
- waiting the short time for a page load
- choosing the latest version

The part about realizing I'm looking at the wrong thing is what takes the longest, and usually only happens when I already spent some time looking at whatever else I came looking for, 

I'd love if there was a small banner telling me that I'm in fact not looking at the latest version. It does not have to be big or very noticeable - something small and unobtrusive like `docs.rs` does would be amazing.

**Comment by LawnGnome** [comment]
Closing in favour of #10710, which is the same thing but with a bit more detail.
